internal/testutil/pkgdata.go

```diff
@@ -1,152 +1,536 @@
 package testutil
 
 import (
-	"encoding/base64"
+	"archive/tar"
+	"bytes"
+	"embed"
+	"path/filepath"
+	"time"
+
+	"github.com/blakesmith/ar"
+	"github.com/klauspost/compress/zstd"
 )
 
 var PackageData = map[string][]byte{}
 
 func init() {
-	baseFilesData, err := base64.StdEncoding.DecodeString(baseFilesBase64)
+	for name, entries := range pkgs {
+		for i, _ := range entries {
+			if !entries[i].noFixup {
+				fixupEntry(&entries[i])
+			}
+		}
+		data, err := buildDeb(entries)
+		if err != nil {
+			panic(err)
+		}
+		PackageData[name] = data
+	}
+}
+
+type tarEntry struct {
+	header  tar.Header
+	noFixup bool
```

Comment:
what is this actually used for? I don't think is it used in this `base-files` test